meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,177 @@
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after sa new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Experimental - Hybrid search improvements
+
+For a future stabilization, multiple changes have been applied to hybrid search features.
+
+📖 More details about the usage in this [page](https://www.notion.so/meilisearch/v1-11-AI-search-changes-0e37727193884a70999f254fa953ce6e).
+
+Done by @dureuill in #4906, #4920, #4892, and #4938.
+
+### ⚠️ Breaking changes
+
+* When using the semantic or the hybrid search, `hybrid.embedder` is now a **mandatory** parameter in `GET and POST /indexes/{:indexUid}/search`
+* As a consequence, it is now **mandatory** to pass `hybrid` even for full-vector search (with only `vector` and not `q`)
+* `embedder` is now a **mandatory** parameter in `GET and POST /indexes/{:indexUid}/similar`
+* Ignore non-zero `semanticRatio` when `vector` is passed but not `q`: a semantic search will be performed.
+
+### Addition & improvements
+
+* The default model for OpenAI is now `text-embedding-3-small` instead of `text-embedding-ada-002`.
+* Limit the maximum length of a rendered document template: when the source of an embedder is set to `huggingFace`, `openAi`, `rest` or `ollama`, then `documentTemplateMaxBytes` is now available as an optional parameter. This parameter describes the number of bytes in which the rendered document template text should fit when trying to embed a document. Longer texts are truncated to fit.
+* Add the ability to query whether a field is searchable in `documentTemplate`: use `field.is_searchable: true` if the field is a searchable attribute, otherwise false.
+* The new document template changed to
+  * filter out fields that are not searchable
+  * filter out fields that are nil or missing for the current document
+
+v1.10 and before:
+```
+{% for field in fields %}
+{{ field.name }}: {{ field.value }}\n
+{% endfor %}
+```
+
+From v1.11:
+```
+{% for field in fields %}
+{% if field.is_searchable and not field.value == nil %}
+{{ field.name }}: {{ field.value }}\n
+{% endif %}
+{% endfor %}
+```
+
+This last change is backward compatible. Embedders that use the v1.10 document template will keep using it. The new default document template will only be applied to newly added embedders.
+
+## Speed up indexing of your vector database with binary quantization
+
+TBD
```

Comment:
````suggestion
## Speed up indexing of your vector database with binary quantization

The binary quantization converts your embeddings of floating point numbers (using 32 bits) into embeddings of boolean values (using 1 bit).
In our benchmarks, it reduced the size of the database by a factor of 10, and divided the indexing time by a factor of 6 without much impact on the search time.

> [!WARNING]  
> Enabling this feature will update all of your vectors to contain only `1`s or `-1`s, impacting the relevancy significantly.
> Once enabled, **there is no coming back**. Before trying it out on your main index, you should try the feature on a smaller index to ensure it fits your use case.

Although this feature degrades the relevancy of the semantic searches, it showed great results on collections with many dimensions (> 1300) and many points (more than 100000).

Enable the binary quantization by updating the settings of your embedder with:

TBD

````

Reply:
Thank you tamo, waiting for your update before approving then 😊

Reply:
I think we're good